helm/common/Makefile

```diff
@@ -0,0 +1,14 @@
+name        	:=      $(shell yq '.name' Chart.yaml)
+version     	:=      $(shell yq '.version' Chart.yaml)
+changes         :=      $(shell yq '.annotations."artifacthub.io/changes"' Chart.yaml)
+changelog       ?=      CHANGELOG.md
+
+lint: 
+	helm lint . --debug --strict
+
+dry-run:
+	helm install $(name)-dry-run . --dry-run --namespace test
+
+.SILENT:
+changelog:
+	touch $(changelog); echo -e "$(version):\n------" >> $(changelog) && echo -e $(changes) >> $(changelog)
```

Comment:
update to add new changes to top of ${changelog} -- so far it's been tricky escaping sequences yq'd from Chart.yaml in order to get this to work